### PR TITLE
Including tx fee into block rewards

### PR DESCRIPTION
**Pull request by ShengguangXiao**
#### What kind of PR is this?:

/kind fix

#### What this PR does / why we need it:

Tx fees not included in the block rewards

#### Which issue(s) does this PR fixes?:

Fixes # 

https://github.com/cakedefi/defichain-private/issues/681

### CODE REVIEW

File: src/miner.cpp [hunk 1]
```diff
@@ -239,7 +239,7 @@ std::unique_ptr<CBlockTemplate> BlockAssembler::CreateNewBlock(const CScript& sc
         coinbaseTx.vout.resize(2);
 
         // Explicitly set miner reward
-        coinbaseTx.vout[0].nValue = CalculateCoinbaseReward(blockReward, consensus.dist.masternode);
+        coinbaseTx.vout[0].nValue = nFees + CalculateCoinbaseReward(blockReward, consensus.dist.masternode);
```
Comment (bvbfan) [inline src/miner.cpp:242]: It should be covered by hardfork hiehgt

**Review by bvbfan — CHANGES_REQUESTED** [review]

File: src/miner.cpp (hunk 1 above)
Comment (bvbfan) [reply src/miner.cpp:242]: If we go without hardfork, newer nodes will generate bigger reward than older one, It's inconstancy.

Comment (ShengguangXiao) [reply src/miner.cpp:242]: I will make the change in another PR based on 1.9.x

**Comment by ShengguangXiao** [comment]
Close this in favor of https://github.com/DeFiCh/ain/pull/764

**Pull request closed without merging**